pkg/secrethub/credentials/encoding.go

```diff
@@ -168,8 +168,8 @@ func newParser(decoders []Decoder) parser {
 }
 
 // parse parses a credential string.
-func (p parser) parse(raw string) (*encodedCredential, error) {
-	parts := strings.Split(raw, ".")
+func (p parser) parse(raw []byte) (*encodedCredential, error) {
+	parts := strings.Split(string(raw), ".")
```

Comment:
If you're doing this, you can also just use `bytes.Split` and avoid the uncessary casts here.

Reply:
Did not know about this function. Let's use it.